lib/internal/process/per_thread.js

```diff
@@ -223,6 +229,49 @@ function wrapProcessMethods(binding) {
     return true;
   }
 
+  function execve(execPath, args, env) {
+    const { isMainThread } = require('internal/worker');
+
+    if (!isMainThread) {
+      throw new ERR_WORKER_UNSUPPORTED_OPERATION('Calling process.execve');
+    } else if (process.platform === 'win32') {
+      throw new ERR_FEATURE_UNAVAILABLE_ON_PLATFORM('process.execve');
```

Comment:
the docs say this is also unavailable on Android. Should that be checked here also?

Reply:
I would remove Android from the docs. It's not a supported platform.

Reply:
Done, thanks!